deployment_report/templates/viya_deployment_report.html.j2

```diff
@@ -130,6 +130,26 @@
   </div>
   {# Cluster Overview: Kubernetes Versions Accordion #}
 
+  {# Cluster Overview: Storage Accordion #}
+  {% set kind_name = "Storage" %}
+  <div class="jq-accordion">
+    <h2>Storage</h2>
+    <div>
+      {% for db, info in report_data.kubernetes.dbInfo.items() %}
+      <h7>Database({{db}})</h7>
+      <table>
+      {% for key, value in info.items() %}
+        <tr>
+          <th>{{key}}</th>
```

Comment:
You can convert keys to displayable values here with: `{{ key | title }}`.

Also, in general, variables should be formatted like `{{ <var_name> }}` (i.e., a space between the var_name and the brackets) for consistency with the rest of the file.